modules/core/shared/src/main/scala/gem/enum/GnirsDecker.scala

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2016-2017 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gem
+package enum
+
+import cats.instances.string._
+import cats.syntax.eq._
+import gem.util.Enumerated
+
+/**
+ * Enumerated type for GNRIS Decker.
+ * @group Enumerations (Generated)
+ */
+sealed abstract class GnirsDecker(
+  val tag: String,
+  val shortName: String,
+  val longName: String,
+  val obsolete: Boolean
+) extends Product with Serializable
+
+object GnirsDecker {
+
+  /** @group Constructors */ case object Acquisition extends GnirsDecker("Acquisition", "ACQ", "Acquisition", false)
+  /** @group Constructors */ case object PupilViewer extends GnirsDecker("PupilViewer", "PV", "Pupil viewer", false)
+  /** @group Constructors */ case object ShortCamLongSlit extends GnirsDecker("ShortCamLongSlit", "SHORT_CAM_LONG_SLIT", "Short camera long slit", false)
```

Comment:
The short name is intended to be seen by users so `SHORT_CAM_LONG_SLIT` is probably not what we want here. If you want to include an internal name I think it should be another field.